styles/typography/typography-preset-5.json

```diff
@@ -0,0 +1,36 @@
+{
+	"version": 3,
+	"$schema": "https://schemas.wp.org/trunk/theme.json",
+	"title": "Literata & Ysabeau Office",
+	"slug": "typography-preset-5",
+	"styles": {
+		"typography": {
+			"fontFamily": "var:preset|font-family|ysabeau-office",
+			"fontSize": "var:preset|font-size|medium",
+			"fontStyle": "normal",
+			"fontWeight": "400",
+			"letterSpacing": "-0.24px",
+			"lineHeight": "1.4"
+		},
+		"blocks": {
+			"core/site-title": {
+				"typography": {
+					"fontFamily": "var:preset|font-family|literata"
```

Comment:
In the design, the site title for this preset is using Ysabeau Office, 20px uppercased. When applying I see Literata at `22px`.